src/ui/components/ShowMoreCard/index.js

```diff
@@ -129,21 +130,26 @@ export class ShowMoreCardBase extends React.Component<InternalProps> {
     invariant(id, 'The id property is required');
 
     const readMoreLink = (
-      <a
-        aria-label={i18n.gettext('Expand to read more')}
+      <Button
```

Comment:
This is a bigger change. Can we do without it for now?

Reply:
I figured it out! I had set it to `props.ariaLabel` instead of just `ariaLabel`.

Reply:
Can we not use `Button` in this patch?